### PR TITLE
fix(build): types

### PR DESCRIPTION
**Pull request by alexx855**

### CODE REVIEW

**Comment by alexx855** [comment]
issue https://github.com/makimenko/angular-template-for-threejs/issues/359

**Comment by makimenko** [comment]
@alexx855 thanks a lot for a contribution! Checked and can't understand why this issue was not detected by CI.

File: projects/atft/src/lib/raycaster/raycaster-group.directive.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as THREE from 'three';
 
 export interface RaycasterEmitEvent {
   component: AbstractObject3D<any>;
-  face?: THREE.Face;
+  face?: THREE.Face3;
```
Comment (makimenko) [inline projects/atft/src/lib/raycaster/raycaster-group.directive.ts:9]: Isn't removed in r126 from core? 
https://github.com/mrdoob/three.js/releases/tag/r126

**Comment by alexx855** [comment]
@makimenko thank  to you! I'm not sure either, it should be detected by the CI.

**Pull request merged**